modules/storages/app/common/storages/errors.rb

```diff
@@ -31,5 +31,9 @@
 module Storages
   module Errors
```

Comment:
🟢 One thing that came to me while looking at this part is that we could have one `Error` (or `BaseError` as RoR loves `BaseSomething` names) and make our other errors inherit from it.

Then we can just rescue our own errors by going with `rescue Storages::Errors::BaseError` and let other more esoteric errors bubble up. WDYT?

Reply:
Sounds like a plan to me. =)